agent/util-scripts/pbench-tool-meister-stop

```diff
@@ -103,20 +181,38 @@ def main(_prog, cli_params):
         full_hostname = os.environ["_pbench_full_hostname"]
         benchmark_run_dir = Path(os.environ["benchmark_run_dir"]).resolve(strict=True)
     except Exception:
-        logger.exception("failed to fetch parameters from the environment")
+        logger.exception("failed to fetch required parameters from the environment")
         return 1
```

Comment:
`-start` returns a variety of error codes...why doesn't `-stop` return the same ones for the same cases?  Moreover, there are two distinct flavors of "failure":  the kind where the tool couldn't even try to do its job (bad tool group, bad command arguments, et al.) and the kind where it tried but something prevented it from working.  It's worth having the status return distinguish at least those two cases.

Reply:
Tracked in https://github.com/distributed-system-analysis/pbench/issues/2177.